scripts/single-node.sh

```diff
@@ -10,7 +10,13 @@ celestia-appd init $CHAINID --chain-id $CHAINID
 celestia-appd keys add validator --keyring-backend="test"
 # this won't work because the some proto types are decalared twice and the logs output to stdout (dependency hell involving iavl)
 celestia-appd add-genesis-account $(celestia-appd keys show validator -a --keyring-backend="test") $coins
-celestia-appd gentx validator 5000000000utia --keyring-backend="test" --chain-id $CHAINID
+celestia-appd gentx validator 5000000000utia \
+  --keyring-backend="test" \
+  --chain-id $CHAINID \
+  --orchestrator-address $(celestia-appd keys show validator -a --keyring-backend="test") \
+  #da6ed55cb2894ac2c9c10209c09de8e8b9d109b910338d5bf3d747a7e1fc9eb9
+  --ethereum-address 0x966e6f22781EF6a6A82BBB4DB3df8E225DfD9488
+
```

Comment:
@YazzyYaz just an fyi that this change requires validators to add two extra fields when they create a validator. This differs from the normal cosmos-sdk, so if we have docs for creating a validator, we will need to update them.

also @SweeXordious we also want to make a note of this somewhere to make sure that we document this in our own docs.

Reply:
We should decide on which docs do we want to write for now. Do we want to wait until the orchestrator/relayer are ready and document everything? or we want to update existing validator docs?

Reply:
@evan-forbes does this apply to Mamaki users or the new Arabica testnet?

Reply:
> does this apply to Mamaki users or the new Arabica testnet?

this will apply to whichever chains will be running using v0.7.0 of celestia-app.

Reply:
OK cc @sysrex @Bidon15 just a heads up for when version 0.7.0 of celestia-app is released, we will need to update the configs on running Arabica.

Reply:
https://github.com/celestiaorg/docs/issues/222

Reply:
Bumping our discussion, 
What exact rc candidate of `0.7.0` do we want to deploy in arabica for celestia-app?

Reply:
depends on https://github.com/celestiaorg/celestia-node/pull/1147, but if we can't merge that, then I would recommend sticking to v0.7.0-rc-1